cri/v1alpha2/cri.go

```diff
@@ -284,16 +284,23 @@ func (c *CriManager) StopPodSandbox(ctx context.Context, r *runtime.StopPodSandb
 
 	// Teardown network of the pod, if it is not in host network mode.
 	if !hostNet {
-		err = c.CniMgr.TearDownPodNetwork(&ocicni.PodNetwork{
-			Name:         metadata.GetName(),
-			Namespace:    metadata.GetNamespace(),
-			ID:           podSandboxID,
-			NetNS:        sandboxMeta.NetNSPath,
-			PortMappings: toCNIPortMappings(sandboxMeta.Config.GetPortMappings()),
-		})
-		if err != nil {
+		_, err = os.Stat(sandboxMeta.NetNSPath)
+		if err == nil {
+			err = c.CniMgr.TearDownPodNetwork(&ocicni.PodNetwork{
+				Name:         metadata.GetName(),
+				Namespace:    metadata.GetNamespace(),
+				ID:           podSandboxID,
+				NetNS:        sandboxMeta.NetNSPath,
+				PortMappings: toCNIPortMappings(sandboxMeta.Config.GetPortMappings()),
+			})
+			if err != nil {
+				return nil, err
+			}
+		} else if !os.IsNotExist(err) {
 			return nil, err
 		}
+		// If the sandbox has been closed, the corresponding network namespace
```

Comment:
Could we put the comment on the top of `if`? if not, it seems that we delete the code but not comment. 😄